superset/commands/chart/export.py

```diff
@@ -71,12 +73,23 @@ def _file_content(model: Slice) -> str:
         if model.table:
             payload["dataset_uuid"] = str(model.table.uuid)
 
+        # Fetch tags from the database if TAGGING_SYSTEM is enabled
+        if feature_flag_manager.is_feature_enabled("TAGGING_SYSTEM"):
+            tags = (
+                model.tags if hasattr(model, "tags") else []
+            )
+            # Filter out any tags that contain "type:" in their name
+            payload["tag"] = [tag.name for tag in tags if not any(x in tag.name for x in ["type:", "owner:"])]
```

Comment:
Since users can create tags with those strings, could we filter for type, instead? something like:
``` python
payload["tags"] = [tag.name for tag in tags if tag.type == TagType.custom]
```